content/blog/2024/frx/index.md

```diff
@@ -0,0 +1,64 @@
+
+The project is heavily inspired by David Donoho's vision of **Frictionless Research Exchanges** (FRX) as described in [_Data Science at the Singularity_](https://arxiv.org/abs/2310.00865).
+
+In this article, Donoho describes three key pillars for Frictionless Research Exchanges:
+
+> The three initiatives are related but separate; and all three have to come together, and in a particularly strong way, to provide the conditions for the new era. Here they are:
+>
+> - [FR-1: Data] datafication of everything, with a culture of research data sharing. One can now find datasets publicly available online on a bewildering variety of topics, from chest x-rays to cosmic microwave background measurements to uber routes to geospatial crop identifications.
+> - [FR-2: Re-execution] research code sharing including the ability to exactly re-execute the same complete workflow by different researchers.
+> - [FR-3: Challenges] adopting challenge problems as a new paradigm powering scientific research. The paradigm includes: a shared public dataset, a prescribed and quantified task performance metric, a set of enrolled competitors seeking to outperform each other on the task, and a public leaderboard. Thousands of such challenges with millions of entries have now taken place, across many fields.
+
+We considered the landscape of tools and services, and felt that [FR-1] and [FR-2] were already well-served by a variety of tools and services for community workspace infrastructure (e.g., JupyterHub: https://jupyterhub.readthedocs.io), sharable computational environments (e.g., BinderHub: https://binderhub.readthedocs.io), authoring and reading computational narratives (e.g., Jupyter Book: https://jupyterbook.org and MyST: https://mystmd.org), and data I/O tools and standards (e.g., Zarr: https://zarr.readthedocs.io and Intake: https://intake.readthedocs.io). 
+
+However there was a natural missing piece for **[FR-3 Challenges]**, and we could not identify any community-managed infrastructure that facilitated data challenges. This is the goal of `frx-challenges`.
+
+## Why facilitate data challenges?
+
+Data challenges are harder than you think! While it is simple enough to run somebody else's code locally, data challenges require a systematic, secure, and automated approach to accepting and evaluating submissions in a fair and repeatable way. Here are some of the big challenges to tackle:
+
+- **Submissions must retain user and team identity**, which means that we must keep track of users and their submissions over time, since data challenges are designed to encourage iterative improvement and optimization.
+- **Evaluations must use potentially complex resources and data** since many data challenges operate by publicly sharing a small dataset, and then running it against a much more complex dataset.
+- **Evaluations must be totally secure**, so that submissions can't do nefarious things like mine cryptocurrency or gain access to the challenge's private data.
```

Comment:
```suggestion
- **Evaluations must be totally secure**, so that submissions can't do nefarious things like mine cryptocurrency or exfiltrate the challenge's private data.
```

Reply:
Since the submission must actually have access to the challenge's private data, but not actually be able to take the data itself back